src/Illuminate/Container/Container.php

```diff
@@ -162,6 +162,17 @@ public function isAlias($name)
     }
 
     /**
+     * Determine if a given string is an alias abstract.
+     *
+     * @param  string  $name
+     * @return bool
+     */
+    public function isAliasAbstract($name)
+    {
+        return isset($this->aliasAbstracts[$name]);
```

Comment:
I'm not sure this is necessary. I think that the abstract aliases should be an implementation detail not a publicly exposed thing.

You won't be able to write a test against the property but I don't think that's a bad thing. The implementation could then be changed or entirely removed if a better solution arises in the future.